### DISCUSSION TITLE
Jaeger Operator Helm chart support for 1.30+

### DISCUSSION THREAD

**Discussion by matheushent** [jaegertracing/jaeger#6165] [General]
Hello everyone.

According to the [Jaeger Operator Helm chart compatibility](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/COMPATIBILITY.md), it officially supports k8s version up to 1.29. Is there any plan to support the newest ones, e.g. 1.30 and 1.31?

**Comment by yurishkuro** [comment]
cc @jaegertracing/helm-charts-maintainers

**Comment by batazor** [comment]
any PR is welcome
